multispec.yaml

```diff
@@ -0,0 +1,20 @@
+version: 1
+
+specs:
+  distroinfo:
+    fedora:
+      distros:
+        - fedora-26-x86_64
+  variant:
+    upstream:
+      image_name: modularitycontainers/tools
+      readme_description: |-
+        [![Build Status](https://travis-ci.org/container-images/tools.svg?branch=master)](https://travis-ci.org/container-images/tools)
+
+        There is a detailed information about this image available in its [help
+        page](https://github.com/container-images/tools/blob/master/help/help.md).
+    fedora-downstream:
+      image_name: f26/tools
```

Comment:
I'm starting to think that this is something that we have to handle better. Basically, I think this value needs to be dependent on current distro. If you had multiple Fedora's (or a CentOS distroinfo), this wouldn't work that good. I'll try to come up with a good solution that won't complicate the multispec format too much. WDYT about this?

Reply:
FWIW, I submitted a PR for distgen that I think could solve this - see https://github.com/devexp-db/distgen/pull/41

Reply:
I completely agree with you.

Reply:
The PR was merged and released as part of distgen 0.16 - feel free to give it a try.